docs/README.md

```diff
@@ -0,0 +1,18 @@
+# How to  run the docs locally
+
+1. <b>(Skip this step if you have already installed hydrus)</b> <br>
+&nbsp;i) Clone the hydrus repository: ```git clone https://github.com/HTTP-APIs/hydrus.git```<br>
```

Comment:
Why are we installing hydrus to build the docs?

Reply:
I am sorry this is a blunder on my side I will remove the hydrus installation part. I must not have paid proper attention